cypress/e2e/signIn.cy.js

```diff
@@ -12,8 +14,8 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
```

Comment:
Ensure that the `db:clear` task is defined in your Cypress task configuration. This task should clear the database to ensure a clean state before each test.